Source/CoreClasses/Game.swift

```diff
@@ -8,21 +8,23 @@
 import Foundation
 
 ///Represents a digital game.
-public class Game {
+public class Game: NSObject {
 
-  public var appID: String
-  public var name: String
-  public var price: Double = 0.00
+  @objc public var appID: String
+  @objc public var name: String
+  @objc public var price: Double = 0.00
 
-  public init(gameName name: String, gamePrice price: Double) {
+  @objc public init(gameName name: String, gamePrice price: Double) {
 
     self.name = name
     self.price = price
     appID = ""
+    
+    super.init()
 
   }
 
-  public init(appid: String, name: String) {
+  @objc public init(appid: String, name: String) {
 
     appID = "\(appid)"
```

Comment:
I don't think you require two public initializers when you can instead have default param values like so:

public init(appId: String = "", price: Double = 0.00, name: String)

This lets you have one init where you set all your values and when you call the init you only need to pass the values that aren't optional